src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -378,8 +378,8 @@ protected function mergeWithExisting($entitiesByModule)
             // Merge
             if ($existingMessages) {
                 $entitiesByModule[$module] = array_merge(
-                    $existingMessages,
-                    $messages
+                    $messages,
```

Comment:
There is a very good case for adding a merge mode parameter. :) Do you want to suggest one?

merge=0 (clear and replace all) (current merge=false)
merge=1 (update and add) (current merge=true)
merge=2 (add new only)

As an example.